[ONNX] use consistent quoting for string literals (#57757)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #58697 [ONNX] Enable support for roll() op. (#58389)
* #58696 [ONNX] handle aten::_set_item on Dict in convertInplaceOpsAndTrackAlias (#58317)
* **#58695 [ONNX] use consistent quoting for string literals (#57757)**
* #58694 [ONNX] Improve lower tuples and handle control flow  (#57650)
* #58693 [ONNX] Update special post process for SequenceInsert after SequenceEmpty (#56965)
* #58692 Support symbolic for conv_tbc (#58359)
* #58691 [ONNX] RNN scripting (#57564)
* #58690 [ONNX] Update instance_norm2 symbolic to handle track_running_stats=True (#55051)

As PEP8 says: "Pick a rule and stick to it." [1]

[1] https://www.python.org/dev/peps/pep-0008/#string-quotes

Co-authored-by: Gary Miguel <garymiguel@microsoft.com>

Differential Revision: [D28714811](https://our.internmc.facebook.com/intern/diff/D28714811)

garymm committed:
[ONNX] use consistent quoting for string literals (#57757)

As PEP8 says: "Pick a rule and stick to it." [1]

[1] https://www.python.org/dev/peps/pep-0008/#string-quotes

Co-authored-by: Gary Miguel <garymiguel@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] use consistent quoting for string literals (#57757)"


As PEP8 says: "Pick a rule and stick to it." [1]

[1] https://www.python.org/dev/peps/pep-0008/#string-quotes

Co-authored-by: Gary Miguel <garymiguel@microsoft.com>

[ghstack-poisoned]

SplitInfinity commented:
@splitinfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28622444).

SplitInfinity commented:
@SplitInfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28714811).

BowenBao committed:
Update on "[ONNX] use consistent quoting for string literals (#57757)"


As PEP8 says: "Pick a rule and stick to it." [1]

[1] https://www.python.org/dev/peps/pep-0008/#string-quotes

Co-authored-by: Gary Miguel <garymiguel@microsoft.com>

Differential Revision: [D28714811](https://our.internmc.facebook.com/intern/diff/D28714811)

[ghstack-poisoned]

SplitInfinity commented:
@SplitInfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28714811).
